src/App.js

```diff
@@ -9,6 +9,20 @@ export class App extends Component {
     movies: data,
```

Comment:
Let's use functional components and hooks instead of classes (as task requires):

> Don't use class components. Use React Hooks instead.